docker/k8s-sglang-service.yaml

```diff
@@ -35,10 +35,20 @@ spec:
           env:
             - name: HF_TOKEN
               value: <secret>
+            - name: NUMBA_CACHE_DIR
```

Comment:
Is it possible that it is not necessary? We should ensure the simplicity of the startup method, especially for Kubernetes. Moreover, users of Kubernetes will inevitably modify these YAML files. This is not the focus of the sglang documentation. I agree with your modification regarding the SHM part. As for other non-root scenarios, users should refer to the Kubernetes community.

Reply:
It is not necessary, when running as root. I can remove it from this PR since it is not related to the original issue